docs/nodejs.md

```diff
@@ -0,0 +1,250 @@
+<!--
+title: Node.js SDK
+menuText: Node.js SDK
+description: 
+menuOrder: 4
+-->
+
+# Node.js SDK
+
+Serverless Console, when Tracing is enabled on an AWS Lambda function, will hook
+into the AWS Lambda runtime environment and automatically report Traces and
+Spans. To capture handled errors, warnings, and to set custom tags, the SDK
+library must be added and instrumented in your AWS Lambda function handler.
+
+## Key terms
+
+- A **Captured Error** is one instance of an error that is sent to Serverless
+Console. It can be viewed in Dev Mode or the Trace Explorer Details.
+- A **Captured Warning** is one instance of a string in Node.js that is sent to
+Serverless Console, much like a Captured Error.
+- A **Tag** is a key/value-pair that can be set on the Trace or an individual
+Captured Error or Captured Warning, and sent to Serverless Console. Tags can be
+viewed on the Trace Explorer Details and Dev Mode.
+
+## Compatibility
+
+While Serverless Console is developed by the makers of the Serverless Framework,
+the entire Serverless Console product and this SDK are 100% agnostic of the
+deployment tool you use. Serverless Console and this SDK work just as well with
+Terraform, CDK, SAM, Pulumi, etc, as as they do with Serverless Framework.
+
+## Installation
+
+**Install the package**
+
+When Tracing is enabled in Servelress Console, an  AWS Lambda Layer is added to
+your AWS Lambda function with the `@serverless/sdk` package. If you are running
+the handler with Tracing enabled on AWS Lambda only, then you can skip this
+step.
+
+If you will run this on AWS Lambda without Serverless Console Tracing enabled,
+or running it on a different runtime, like locally, then you'll need to add the
+`@serverless/sdk` package.
+
+```
+npm install @serverless/sdk --save
+# or
+yarn add @serverless/sdk
+```
+
+**Using a bundler**
+
+If you use a bundler, like esbuild, the AWS Lambda Layer for Serverless Console
+will not be able to auto-instrument traces and spans on your handler. To enable
```

Comment:
The above message may confuse users that instrumentation won't work at all.

Handler and native Node.js APIs (like `http` or `console`) remain instrumented. It's just dependencies that are bundled that will not. So at this point, it applies only to `express` and AWS SDK (assuming the user relies on the bundled version).